package.json

```diff
@@ -8,7 +8,8 @@
     "husky": "^1.0.0",
     "lint-staged": "^7.3.0",
     "react": "^16.5.2",
-    "react-dom": "^16.5.2"
+    "react-dom": "^16.5.2",
+    "react-helmet": "^5.2.0"
```

Comment:
Is there a reason for having this package other than to change the page title? If that's the only reason, then I don't think we need it, since we can edit `public/index.html`